src/libecalc/common/errors/exceptions.py

```diff
@@ -98,3 +103,20 @@ def __init__(self, header: str):
         super().__init__(
             "Missing column", f"Column matching header '{header}' is missing.", error_type=EcalcErrorType.CLIENT_ERROR
         )
+
+
+class InvalidColumn(InvalidResource):
+    def __init__(self, header: str, message: str, row: int = None):
+        self.header = header
+        self.row = row
+        super().__init__(
+            "Invalid column",
```

Comment:
possible to indicate which column that is invalid?

Reply:
It's handled in the message currently. Not sure what to make of the title in these errors vs message.